### DISCUSSION TITLE
wcautil.h is missing

### DISCUSSION THREAD

**Discussion by vladimir-andreevich** [nodejs/help#4699] [Q&A] [answered]
### Version

v14.19.3

### Platform

Microsoft Windows NT 10.0.19044.0 x64

### What steps will reproduce the bug?

1. Open `tools/msvs/msi/nodemsi.sln`
2. Build a WiX solution using a local Windows debugger for the target x64 or x86.

### What is the expected behavior?

Visual Studio IDE makes WiX Node.JS installers for Windows.

### What do you see instead?

Error code C1083 in custom_actions.cc, line 5: Cannot include file `wcautil.h`: No such file or directory.

### Additional information

WiX toolset v3.11

**Comment by StefanStojanovic** [comment, marked as answer]
The Node.js installer is built from `vcbuild.bat` with this command https://github.com/nodejs/node/blob/main/vcbuild.bat#L535. Since you are trying to build it from Visual Studio IDE, MSBuild is not getting all of the required properties. I suggest you try building the MSI installer by running `vcbuild.bat release x64 msi`

**Reply by vladimir-andreevich** [reply]
I think this note shall be added to `BUILDING.md` so other developers can immediately see how to build Windows MSI installers.
